### PR TITLE
#286 Implement layout menu toggler in sidebar

### PR DESCRIPTION
**Pull request by jakedcolbert**
Add a drawer menu option to display the layout option menu. Using the collectionSelector id, call the showModal() function to show the menu.

This is achieved by: 
- Adding a button and divider to the drawer with a translated label
- Adding visibility functionality by including the option if the `config.mainFeatures['layout-config-change-nav-drawer-menu']` configuration is true.
- Adding an `onclick` handler to the button to execute "collectionSelector.showModal()"

Implements #286

### CODE REVIEW

**Comment by jakedcolbert** [comment]
Ready for review @chrisvire

**Comment by chrisvire** [comment]
image

**Review by chrisvire — CHANGES_REQUESTED** [review]
Fix the `&amp;` displaying

File: src/lib/components/Sidebar.svelte [hunk 1]
```diff
@@ -86,6 +89,20 @@ The sidebar/drawer.
...
                 </picture>
             </a>
+            {#if showLayouts}
+                <li>
+                    <!-- svelte-ignore a11y-missing-attribute -->
+                    <button
+                        style:color={textColor}
+                        style:direction={$direction}
+                        class="btn"
+                        onclick="collectionSelector.showModal()"
```
[the remark this answers is not in the document]
Comment (jakedcolbert) [reply src/lib/components/Sidebar.svelte]: @chrisvire

[the remark this answers is not in the document]
Comment (chrisvire) [reply src/lib/components/Sidebar.svelte]: Yeah, that is not good.  We need to figure something else out.

**Comment by jakedcolbert** [comment]
All I did was rebase. Help!

**Comment by jakedcolbert** [comment]
All I did was rebase. Help!!

**Pull request closed without merging**